articles/connectivity/conn-faq.md

```diff
@@ -19,7 +19,7 @@ toc_mdlink: conn-faq.md
 
 ## What options do I have to connect to my solution hosted on the UKCloud Assured OFFICIAL (previously PGA IL2) cloud platform?
 
-The UKCloud Assured OFFICAL cloud platform previously achieved CESG Pan Government Accreditation (PGA) and is PSN Accredited to host OFFICIAL data (including OFFICIAL SENSITIVE) which is accessible via the internet or the PSN Assured Service. You can also connect via internet, N3/HSCN and Janet dependant on relevant accreditation.
+The UKCloud Assured OFFICAL cloud platform previously achieved CESG Pan Government Accreditation (PGA) and is PSN Accredited to host OFFICIAL data (including OFFICIAL SENSITIVE) which is accessible via the internet or the PSN Assured Service. You can also connect via internet, HSCN and Janet dependant on relevant accreditation.
```

Comment:
can we remove 'Assured' after PSN? i dont know how to do this in review mode...

Reply:
Fixed